docs/source/building-blocks.rst

```diff
@@ -26,7 +26,7 @@ These modules and their helper classes implement the Transformer attention mecha
 .. autoclass:: curated_transformers.layers.attention.QkvMode
    :members:
 
-.. autoclass:: curated_transformers.layers.attention.QkvHeadSharing
+.. autoclass:: curated_transformers.layers.attention.AttentionHeads
```

Comment:
Missing `TransformerDropouts` in the previous section.

Reply:
Added.